src/zone-patch/index.js

```diff
@@ -48,9 +48,9 @@ function wrapTestInZone(testBody) {
   if (testBody === undefined) {
     return;
   }
-  return testBody.length === 0
-    ? () => testProxyZone.run(testBody, null)
-    : done => testProxyZone.run(testBody, null, [done]);
+  return function() {
+    return testProxyZone.run(testBody, null, arguments);
```

Comment:
How about return `(...args) => testProxyZone.run(testBody, null, args)`? Not that it matters too much, it's just my preference of not using `arguments` when possible

Reply:
Sure, I also think that is a good idea, will update it.